finished/build.gradle

```diff
@@ -66,29 +66,26 @@ android {
 
 dependencies {
     // TODO: Review Dependencies
```

Comment:
remove?

Reply:
this is part of the codelab narrative itself where we invite the reader to review the dependencies to udnerstand what they are doing, it's not a classical developer TODO